engine/engine/content/builtins/scripts/debugger.lua

```diff
@@ -25,9 +25,162 @@ local function onexit(code, close)
   msg.post("@system:", "exit", {code = code})
 end
 
+-- Lightweight hook layer
+
+local mobdebug_hook
+local mobdebug_on
+local breakpoints = {}
+local coroutines = {}; setmetatable(coroutines, {__mode = "k"}) -- "weak" keys
```

Comment:
I think you can write this as:

```
local coroutines = setmetatable({}, {__mode = "k"}) -- "weak" keys
```

Reply:
I keep it the same style as in mobdebug itself to emphasize the logic duplication in this area.

Reply:
Ok, got it!